src/main/java/org/primeframework/mvc/security/BaseUserIdCookieSecurityContext.java

```diff
@@ -238,12 +247,16 @@ private UserIdSessionContext<TUserId> resolveContext() {
       this.request.setAttribute(ContextKey, context);
       return context;
     } catch (BadPaddingException e) {
+      logger.debug("User cookie parsing failed. It is likely the cookie encryption key has changed, deleting session",
```

Comment:
We should use complete sentences with punctuation in all of our logger messages.

Reply:
Done